wazo_calld/plugins/adhoc_conferences/http.py

```diff
@@ -28,3 +28,44 @@ def post(self):
         )
 
         return adhoc_conference, 201
+
+
+class UserAdhocConferenceResource(AuthResource):
+
+    def __init__(self, adhoc_conference_service, auth_client):
+        self._adhoc_conference_service = adhoc_conference_service
+        self._auth_client = auth_client
+
+    @required_acl('calld.users.me.conferences.adhoc.create')
```

Comment:
Should it be `calld.users.me.conferences.adhoc.delete`?

Reply:
I fixed it to match the `api.yml` and rebased the branch on `master` to make Zuul faster when you guys get back.

Reply:
Thank you!